tests/arch/arm/arm_irq_vector_table/src/arm_irq_vector_table.c

```diff
@@ -230,6 +230,18 @@ vth __irq_vector_table _irq_vector_table[] = {
 	isr0, isr1, isr2, 0,
 	rtc_isr
 };
+#elif defined(CONFIG_SOC_SERIES_IMX_RT6XX)
+/* MXRT685 employs a OS Event timer to implement the Kernel system
+ * timer, instead of the ARM Cortex-M SysTick. Therefore, a pointer to
+ * the timer ISR needs to be added in the custom vector table to handle
+ * the timer "tick" interrupts.
+ */
+extern void mcux_lpc_ostick_isr(void);
```

Comment:
I'll put this out as a suggestion as others may disagree with the approach.  In mcux_os_timer.c I would leave mcux_lpc_ostick_isr alone and add
void test_mcux_lpc_ostick_isr(void)
{
    mcux_lpc_ostick_isr(NULL);
}

Then _irq_vector_table can reference test_mcux_lpc_ostick_isr().  This solves the problem where you are implicitly casting the "void *" to "void" with the extern declaration of mcux_lpc_ostick_isr here.

Reply:
Unless it is absolutely necessary, we should not be adding test-related code to the driver itself; if any, `test_mcux_lpc_ostick_isr` should be implemented on the test-side (i.e. in `arm_irq_vector_table.c`).